gitlab/objects.py

```diff
@@ -935,7 +935,7 @@ class ProjectHookManager(BaseManager):
 class ProjectIssueNote(GitlabObject):
     _url = '/projects/%(project_id)s/issues/%(issue_id)s/notes'
     _constructorTypes = {'author': 'User'}
-    canUpdate = False
+    canUpdate = True
```

Comment:
You can remove the line (canUpdate defaults to True in the GitlabObject class).

Otherwise looks good.